### PR TITLE
feat(NeuralProfile): Added 3 icons, HourGlassIcon, InfluencerIcon2, P…

### PR DESCRIPTION
**Pull request by johansonis**
…honeFilledIcon2

### CODE REVIEW

File: src/components/icons/HourGlassIcon.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const HourGlassIcon = props => (
+    <svg {...props.size || { width: '15px', height: '25px' }} {...props} viewBox="0 0 15 25">
+        {props.title && <title>{props.title}</title>}
+        <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```
Comment (MorganNesbitt) [inline src/components/icons/HourGlassIcon.js]: these arent proper react svg attributes. react svg attributes are camel case

File: src/components/icons/PhoneFilledIcon2.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const PhoneFilledIcon2 = props => (
+    <svg {...props.size || { width: '24px', height: '24px' }} {...props} viewBox="0 0 24 24">
+        {props.title && <title>{props.title}</title>}
```
Comment (MorganNesbitt) [inline src/components/icons/PhoneFilledIcon2.js]: same here

File: src/components/icons/HourGlassIcon.js (hunk 1 above)
Comment (johansonis) [reply src/components/icons/HourGlassIcon.js]: fixed

File: src/components/icons/PhoneFilledIcon2.js (hunk 2 above)
Comment (johansonis) [reply src/components/icons/PhoneFilledIcon2.js]: fixed

**Pull request merged**